packages/dd-trace/test/config.spec.js

```diff
@@ -261,6 +261,9 @@ describe('Config', () => {
     expect(config).to.have.nested.property('installSignature.id', null)
     expect(config).to.have.nested.property('installSignature.time', null)
     expect(config).to.have.nested.property('installSignature.type', null)
+    expect(config).to.have.nested.property('llmobs.mlApp', undefined)
+    expect(config).to.have.nested.property('llmobs.agentlessEnabled', false)
+    expect(config).to.have.nested.property('llmobs.apiKey', undefined)
```

Comment:
can we also add tests for llmobs.enabled?

Reply:
I added some tests for `llmobs.enabled` at the bottom of the config tests. but, i wanted to set `config.llmobs.enabled` with `DD_LLMOBS_ENABLED` overriding the existence of `options.llmobs`, ie environment overrides options for that one config property. with regard to the other changes you've proposed, that's why I had it originally computing `config.llmobs.enabled` in the constructor by coalescing the environment variable first, and then the existence of the options to compute `llmobs.enabled`. With regards to the tests at the bottom, and that desired behavior, is there a way I can do that in the config? it's my understanding that `options` will always override `environment`, which is what i don't want in this case for `llmobs.enabled`.